README.md

```diff
@@ -1,53 +1,53 @@
-# Holochain
-
-[![Project](https://img.shields.io/badge/project-holochain-blue.svg?style=flat-square)](http://holochain.org/)
-[![Forum](https://img.shields.io/badge/chat-forum%2eholochain%2enet-blue.svg?style=flat-square)](https://forum.holochain.org)
-[![Chat](https://img.shields.io/badge/chat-chat%2eholochain%2enet-blue.svg?style=flat-square)](https://chat.holochain.org)
-
+[![Project](https://img.shields.io/badge/Project-Holochain-blue.svg?style=flat-square)](http://holochain.org/)
+[![Forum](https://img.shields.io/badge/Forum-forum%2eholochain%2enet-blue.svg?style=flat-square)](https://forum.holochain.org)
+[![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
+![Test](https://github.com/holochain/holochain-client-js/actions/workflows/test.yml/badge.svg?branch=main)
 [![Twitter Follow](https://img.shields.io/twitter/follow/holochain.svg?style=social&label=Follow)](https://twitter.com/holochain)
-License: [![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
 
-This repository contains the core Holochain libraries and binaries.
+# Holochain
 
-This is the most recent and well maintained version of Holochain with a refactored state model (you may see references to it as Holochain RSM).
+Holochain is an open-source framework to develop peer-to-peer applications with high levels of security, reliability, and performance.
+This repository contains the Holochain core libraries.
 
 ## Code Status
 
-This code is in alpha. It is not for production use. The code is guaranteed NOT secure.
+This code is in alpha. It is not for production use. We will be frequently and heavily restructuring code APIs and data chains until Beta.
```

Comment:
Hmm but I kinda like the guarantee of insecurity. Alan Watts [would be proud](https://www.goodreads.com/book/show/551520.The_Wisdom_of_Insecurity).

But seriously, I think we should leave in a security disclaimer of some kind.

```suggestion
This code is in alpha. It is not for production use, and currently makes no security guarantees. We will be frequently and heavily restructuring code APIs and data chains until Beta.
```

Reply:
@maackle I see. I took it out because I felt it's implicit between all the disclaimers that the code is not secure and secondly I didn't know what "secure" referred to. Okay, let's put it in.